addons/sys_intercom/fnc_infantryPhoneAction.sqf

```diff
@@ -71,7 +71,16 @@ private _infantryPhoneSpeakerAction = [
     localize LSTRING(infantryPhone),
     ICON_RADIO_CALL,
     {true},
-    {_this call FUNC(isInfantryPhoneSpeakerAvailable)},
+    {
+        private _intercomNames = _target getVariable [QGVAR(intercomNames), []];
+        private _intercomAvailable = false;
+        // Find if at least one intercom is available
+        {
+            _intercomAvailable = [_target, acre_player, _forEachIndex] call FUNC(isInfantryPhoneSpeakerAvailable);
```

Comment:
`_player` is already passed, just like `_target`.

Reply:
I like explicit things...

Reply:
Yes, but it's already passed, you get direct object of who did the action (player), so either use `_player` directly or also `params` it above.